<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebRadioDialog</class>
 <widget class="QDialog" name="WebRadioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>play()</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>pause()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>stop()</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WebRadioDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WebRadioDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
